controllers/handler/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -351,7 +351,9 @@ func (r *NodeNetworkConfigurationPolicyReconciler) incrementUnavailableNodeCount
 	}
 	maxUnavailable, err := node.MaxUnavailableNodeCount(r.APIClient, policy)
 	if err != nil {
-		return err
+		r.Log.Info(
+			fmt.Sprintf("failed calculating limit of max unavailable nodes, defaulting to %d, err: %s", maxUnavailable, err.Error()),
+		)
```

Comment:
```suggestion
		r.Log.Error(err, fmt.Sprintf("failed calculating limit of max unavailable nodes, defaulting to %d", maxUnavailable)

```

Reply:
Just a question, this is a recoverable error, do we want to log it as `Error`?
It may be harder then to find real errors in logs

Reply:
I remember even changing some of our logs from error to info just for that reason

Reply:
If it's recoverable is like a warning so is good.